From a79408ca0c2b42d8e811d639d5e66cb88bdcdf24 Mon Sep 17 00:00:00 2001
From: Leonardo Graboski Veiga <leonardo.veiga@toradex.com>
Date: Wed, 6 Nov 2019 09:38:24 -0300
Subject: [PATCH 1/2] pasta-demo: support wayland weston

---
 conf/machine/apalis-imx8.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/conf/machine/apalis-imx8.conf b/conf/machine/apalis-imx8.conf
index c2a2ab4..72ce51e 100644
--- a/conf/machine/apalis-imx8.conf
+++ b/conf/machine/apalis-imx8.conf
@@ -54,7 +54,7 @@ MACHINE_FIRMWARE_append = " firmware-imx-vpu-imx8"
 IMXBOOT_TARGETS = "flash_b0"
 
 PREFERRED_PROVIDER_virtual/libg2d_imxdpu = "imx-dpu-g2d"
-PREFERRED_VERSION_wayland-protocols_mx8 = "1.16.imx"
+PREFERRED_VERSION_wayland-protocols_mx8 = "1.17.imx"
 PREFERRED_VERSION_weston_mx8 = "5.0.0.imx"
 
 BOARD_TYPE = "apalis"
-- 
2.17.1

